.eslintrc.js

```diff
@@ -70,5 +71,6 @@ module.exports = {
     'semi': ['error', 'never'],
 
     '@typescript-eslint/prefer-for-of': 'off',
+    '@typescript-eslint/no-shadow': ['error'], // https://github.com/typescript-eslint/typescript-eslint/issues/2483#issuecomment-687095358
```

Comment:
Ah, this is a good fix that we should propagate to the rest of our repos. Another reason to add a repo for our ESLint rules 😎